nodejs/src/uws.js

```diff
@@ -437,18 +443,16 @@ class Server extends EventEmitter {
             _upgradeReq = null;
         });
 
-        if (options.port) {
-            if (options.host) {
-                this.httpServer.listen(options.port, options.host, ()=> {
-                      this.emit('listening');
-                      callback && callback();
-                });
-            } else {
-                this.httpServer.listen(options.port,  ()=> {
-                      this.emit('listening');
-                      callback && callback();
-                });
-            }
+        if (options.host) {
+            this.httpServer.listen(options.port, options.host, ()=> {
```

Comment:
you could also fix all previous style issues here (6 space indent, () => {}, etc). Prior PR messed it up